app/modules/gpio_pulse.c

```diff
@@ -434,7 +434,7 @@ static int gpio_pulse_start(lua_State *L) {
   pulser->next_adjust = initial_adjust;
 
   // Now start things up
-  if (!platform_hw_timer_init(TIMER_OWNER, FRC1_SOURCE, TRUE)) {
+  if (!platform_hw_timer_init(TIMER_OWNER, FRC1_SOURCE, FALSE)) {
```

Comment:
I don't know why I used autoload=true when I first wrote the code. It wasn't needed. I did make the new hw_timer stuff work with autoload=true before I switched this over to FALSE.